### PR TITLE
docs(alias): add helpful notes

### PR DESCRIPTION
**Pull request by davidrot**
## Rollup Plugin Name: `Alias`

This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [x] documentation
- [x] other

Are tests included?

- [ ] yes (_bugfixes and features will not be merged without tests_)
- [x] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

If yes, then include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking. 

List any relevant issue numbers: -

### Description

Added my thoughts and findings to the readme. Those information would have helped me very much.

### CODE REVIEW

File: packages/alias/README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ import batman from 'batman';
 
 If this seems familiar to Webpack users, it should. This is plugin mimics the `resolve.extensions` and `resolve.alias` functionality in Webpack.
 
+This plugin is working for all kind of files (.ts, .svelte, ...)
```
Comment (shellscape) [inline packages/alias/README.md:30]: What was the intent behind adding this line?

File: packages/alias/README.md [hunk 2]
```diff
@@ -67,6 +69,8 @@ module.exports = {
 
 Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#command-line-reference) or the [API](https://www.rollupjs.org/guide/en/#javascript-api). If the build produces any errors, the plugin will write a 'alias' character to stderr, which should be audible on most systems.
 
+The find input should not end with a slash or it will not work.
```
Comment (shellscape) [inline packages/alias/README.md:72]: Please try rephrasing this. I'm not quite sure what you're trying to speak about here.

**Review by shellscape — CHANGES_REQUESTED** [review]
Thanks for opening a PR. Would like some clarity around your changes.

File: packages/alias/README.md (hunk 1 above)
Comment (davidrot) [reply packages/alias/README.md:30]: I thought this plugin is not compatible with ts-files but this plugin makes the typescript paths feature (https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) kind of obsolet.

File: packages/alias/README.md (hunk 2 above)
Comment (davidrot) [reply packages/alias/README.md:72]: Maybe you can help me, englisch is not my mother tongue. With this phrase I was trying to describe this (https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) check. When I was first trying this plugin it was not working for me. After investigating and debugging what took me a while i saw this check.

File: packages/alias/README.md (hunk 1 above)
Comment (shellscape) [reply packages/alias/README.md:30]:
```suggestion
This plugin will work for any file type that Rollup natively supports, or those which are [supported by third-party plugins](https://github.com/rollup/awesome#other-file-imports).
```

File: packages/alias/README.md (hunk 2 above)
Comment (shellscape) [reply packages/alias/README.md:72]:
```suggestion
```

I'm going to remove this here, and place it in a more appropriate spot.

**Pull request merged**